### PR TITLE
i18n(es): Update `i18n`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- What does this PR change? 
Updated i18n to spanish based on this commit 7f92213a0b93de5a844816841a6bc9cdd371de0c

### CODE REVIEW

File: docs/src/content/docs/es/guides/i18n.mdx [hunk 1]
```diff
@@ -140,7 +140,11 @@ Si no hay una traducción disponible para un idioma, Starlight mostrará a los l
 
 ## Traduce la UI de Starlight
 
-Starlight proporciona de forma predeterminada traducciones para los textos utilizados en la UI al inglés, checo, francés, alemán, italiano, japonés, holandés, portugués, danés, español, turco, árabe, noruego, farsi, hebreo, chino simplificado, coreano, indonesio, ruso, sueco, ucraniano y vietnamita. También damos la bienvenida a [contribuciones para agregar más idiomas predeterminados](https://github.com/withastro/starlight/blob/main/CONTRIBUTING.md).
+Adicionalmente a alojar archivos de contenido traducidos, Starlight te permite traducir las cadenas de UI predeterminadas (p. ej. el encabezado "En esta página" en la tabla de contenidos) para que tus lectores puedan experimentar tu sitio completamente en el idioma seleccionado.
```
Comment (kevinzunigacuellar) [inline docs/src/content/docs/es/guides/i18n.mdx:143]:
```suggestion
Además de alojar archivos de contenido traducidos, Starlight te permite traducir las frases de UI predeterminadas (p. ej. el encabezado "En esta página" en la tabla de contenidos) para que tus lectores puedan experimentar tu sitio completamente en el idioma seleccionado.
```

File: docs/src/content/docs/es/guides/i18n.mdx [hunk 2]
```diff
@@ -140,7 +140,11 @@ Si no hay una traducción disponible para un idioma, Starlight mostrará a los l
...
+
+Inglés, checo, francés, alemán, italiano, japonés, portugués, holandés, danés, español, turco, árabe, noruego, farsi, hebreo, chino simplificado, coreano, indonesio, ruso, sueco, ucraniano, vietnamita y gallego se proporcionan de forma predeterminada, y damos la bienvenida a [contribuciones para agregar más idiomas predeterminados](https://github.com/withastro/starlight/blob/main/CONTRIBUTING.md).
+
+Puedes proprocionar traducciones para idiomas adicionales que admitas, o anular nuestras etiquetas predeterminadas, a través de la colección de datos `i18n`.
```
Comment (kevinzunigacuellar) [inline docs/src/content/docs/es/guides/i18n.mdx:147]:
```suggestion
Puedes proprocionar traducciones para idiomas adicionales que admitas, o editar nuestras etiquetas predeterminadas, a través de la colección de datos `i18n`.
```

**Review by kevinzunigacuellar — COMMENTED** [review]
Gracias @dreyfus92 💜 , deje un par de comentarios.

**Comment by dreyfus92** [comment]
Thank you so much for the review @kevinzunigacuellar 😁

**Pull request merged**